vue/components/ui/organisms/details-expandable/details-expandable.stories.js

```diff
@@ -0,0 +1,64 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs } from "@storybook/addon-knobs";
+
+storiesOf("Components/Organisms/Details Expandable", module)
+    .addDecorator(withKnobs)
+    .add("Details Expandable", () => ({
+        props: {
+            data: {
+                type: Object,
+                default: () => ({
```

Comment:
Use some more generic default like the one from: https://github.com/ripe-tech/ripe-components-vue/blob/v-pca/637-create-details-expandable-component/vue/components/ui/organisms/container-form/container-form.stories.js#L15

Of course you can expand that if need be (pun intended)

Reply:
isn't the one we have default enough?

Reply:
It is shipping specific, when this component won't be specifically for shipping.